ui/keys/keys.go

```diff
@@ -158,3 +159,52 @@ var Keys = KeyMap{
 		key.WithHelp("q", "quit"),
 	),
 }
+
+// Rebind will update our saved keybindings from configuration values.
+func Rebind(universal, issueKeys, prKeys []config.Keybinding) error {
+	err := rebindUniversal(universal)
+	if err != nil {
+		return err
+	}
+
+	err = rebindPRKeys(prKeys)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func rebindUniversal(universal []config.Keybinding) error {
+	for _, kb := range universal {
+		log.Debug("Rebinding key", "builtin", kb.Builtin, "key", kb.Key)
+		switch kb.Builtin {
+		case "up":
+			Keys.Up = kb.NewBinding(&Keys.Up)
+		case "down":
+			Keys.Down = kb.NewBinding(&Keys.Down)
+		case "firstline": // should this be first-line or firstLine?
```

Comment:
I think `firstLine` would be more readable

Reply:
Using camel case for keys in next commit.